src/overlays/actors/ovl_En_Ru1/z_en_ru1.c

```diff
@@ -2349,14 +2350,14 @@ void EnRu1_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot,
     EnRu1* this = (EnRu1*)thisx;
 
     if (limbIndex == RUTO_CHILD_HEAD) {
-        Vec3f vec1;
-        Vec3f vec2;
-
-        vec1 = sMultVec;
-        Matrix_MultVec3f(&vec1, &vec2);
-        this->actor.focus.pos.x = vec2.x;
-        this->actor.focus.pos.y = vec2.y;
-        this->actor.focus.pos.z = vec2.z;
+        Vec3f src;
+        Vec3f dest;
+
+        src = sMultVec;
+        Matrix_MultVec3f(&src, &dest);
+        this->actor.focus.pos.x = dest.x;
+        this->actor.focus.pos.y = dest.y;
```

Comment:
imo "dest" can be `focusPos`.
"src" can also have a better name, but we are currently debating the form these names should take, over here https://github.com/zeldaret/oot/pull/1596#discussion_r1422678368

Reply:
As you can probably guess, I had trouble coming up with names for these two vectors, which is why I settled on `src` and `dest`.
For `src`, it's worth noting that this is the one and only place where `sMultVec` is used, and it runs along the Y-axis.
Is the `focus` vector output from the matrix multiplication supposed to determine where Ruto turns her head to look at Link while standing? `EnRu1_UpdateHeadRotation` only applies when Link is carrying her.

For now, I think I'll call them `focusBase` and `focusPos`, pending a more informed opinion.

Reply:
`focusBase` doesnt really work. What the "base" is, is the matrix at the top of the stack. The "implied" argument that you dont get to name. 
This will be changed someday when we can manage to agree on a convention for it (in that other thread). I would reccomend just naming it after "multvec" for now

Reply:
`multVec` was the other name I considered. I'm aware that the matrix that gets multiplied into the vector is located under the hood, but I don't see anything else in the Child Ruto code that influences that matrix. My best guess was that it's a Y-axis vector because Ruto's head is meant to only turn along that axis, and the matrix in this instance had something to do with the geometry of the Jabu-Jabu dungeon environment.

Reply:
since this is in a `PostLimbDraw` function, the current matrix is the one transforming from the space of the current (head) limb into world space, as computed by the multiplication of [the actor-to-world matrix set by `Actor_Draw`](https://github.com/zeldaret/oot/blob/b37f62fc1fabf75358867fc931591278923288aa/src/code/z_actor.c#L2588-L2592) with the [skelanime system's successive limb transforms](https://github.com/zeldaret/oot/blob/b37f62fc1fabf75358867fc931591278923288aa/src/code/z_skelanime.c#L533)

`sMultVec` is the location of the focus position in the space of the head limb